### PR TITLE
FAI-7530: Upgrade to v1.46.8

### PR DESCRIPTION
**Pull request by eskrm**
https://github.com/metabase/metabase/releases/tag/v1.46.8

This is what I ran, starting from HEAD of the `faros` branch:

```
git checkout -b es/upgrade-v1.46.8
git merge v1.46.8
git checkout --theirs .
git add .
git commit -m "FAI-7530: Upgrade to v1.46.8"
```

### CODE REVIEW

File: .github/workflows/require-percy-tests.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Require Percy specs
```
Comment (tovbinm) [inline .github/workflows/require-percy-tests.yml:1]: @chalenge / @ogusak do we need these GHA workflows or we should disable them?

**Comment by chalenge** [comment]
Lgtm

**Review by tovbinm — COMMENTED** [review]
@eskrm were the any merge conflicts?

**Comment by eskrm** [comment]
> @eskrm were the any merge conflicts?

Yeah, but not many. Didn't look like anything to do with our changes so I took the v.1.46.8 changes to resolve.

File: enterprise/backend/test/metabase_enterprise/advanced_config/file/databases_test.clj [hunk 2]
```diff
@@ -3,14 +3,16 @@
    [clojure.test :refer :all]
    [metabase-enterprise.advanced-config.file :as advanced-config.file]
    [metabase.db.connection :as mdb.connection]
+   [metabase.driver.h2 :as h2]
```
Comment (ogusak) [inline enterprise/backend/test/metabase_enterprise/advanced_config/file/databases_test.clj:6]: I thought h2 was removed due to the earlier security issue?

File: src/metabase/api/table.clj [hunk 3]
```diff
@@ -6,6 +6,7 @@
    [metabase.api.common :as api]
    [metabase.db.query :as mdb.query]
    [metabase.driver :as driver]
+   [metabase.driver.h2 :as h2]
```
Comment (ogusak) [inline src/metabase/api/table.clj:9]: same thing here about h2

**Review by chalenge — APPROVED** [review]
See @ogusak commente

File: src/metabase/driver/h2.clj [hunk 4]
```diff
@@ -34,10 +35,36 @@
...
 (driver/register! :h2, :parent :sql-jdbc)
 
+(def ^:dynamic *allow-testing-h2-connections*
+  "Whether to allow testing new H2 connections. Normally this is disabled, which effectively means you cannot create new
+  H2 databases from the API, but this flag is here to disable that behavior for syncing existing databases, or when
+  needed for tests."
+  ;; you can disable this flag with the env var below, please do not use it under any circumstances, it is only here so
+  ;; existing e2e tests will run without us having to update a million tests. We should get rid of this and rework those
+  ;; e2e tests to use SQLite ASAP.
+  (or (config/config-bool :mb-dangerous-unsafe-enable-testing-h2-connections-do-not-enable)
```
Comment (eskrm) [inline src/metabase/driver/h2.clj:45]: @ogusak @chalenge They didn't remove h2 completely, they just disallow creating new h2 databases. See comment ^.

File: .github/workflows/require-percy-tests.yml (hunk 1 above)
Comment (ogusak) [reply .github/workflows/require-percy-tests.yml:1]: we do not need them and we didn't have them - looking at the history of runs - we only had there our own GHA on merge to faros: https://github.com/faros-ai/metabase/actions?page=2

Comment (ogusak) [reply .github/workflows/require-percy-tests.yml:1]: @eskrm - we should remove all actions except https://github.com/faros-ai/metabase/blob/es/upgrade-v1.46.8/.github/workflows/faros-build.yml

Comment (eskrm) [reply .github/workflows/require-percy-tests.yml:1]: Can we disable them instead? If I remove them they might come back next PR.

**Review by ogusak — APPROVED** [review]

**Pull request merged**